project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/ProteomicsSpecificMeasurementMetadata.java

```diff
@@ -80,4 +80,8 @@ public String fractionName() {
     return fractionName;
   }
 
+  public String comment() {
+    return comment;
+  }
+
```

Comment:
Why was the comment included in the `ProteomicsSpecificMeasurementMetadata`? 
Shouldn't it be in the `ProteomicsMeasurment` class? 

If not, the comments are currently not shown in the measurementgrid itself, since they are collected from the `ProteomicsMeasurement` class, which always returns an empty optional. 🤦